docs/pages/app-signing/apple-team-permissions.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Apple Developer account permissions for EAS Build
```

Comment:
```suggestion
title: Apple Developer Program roles and permissions for EAS Build
```

Based on https://developer.apple.com/support/roles/ I think using "Program" and "Roles" would make this more descriptive